2.4/Dockerfile

```diff
@@ -37,7 +34,6 @@ RUN yum install -y epel-release && \
     yum install -y --setopt=tsflags=nodocs bind-utils gettext iproute v8314 mongodb24-mongodb mongodb24 && \
     yum clean all && \
     mkdir -p /var/lib/mongodb/data && chown -R mongodb.mongodb /var/lib/mongodb/ && \
```

Comment:
This applies to other Dockerfile and other images as well. Also, please remove the `-R`, it is not needed.